language-bindings/rust/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "wamr-sdk-rust"
+version = "0.1.0"
+edition = "2021"
```

Comment:
Is it better to change to 2023?

Reply:
Editions in rust are used to introduce backward incompatible changes without breaking existing projects; a new edition is not created every year. Currently there are three editions, `2015`, `2018`, and `2021`. New crates created by cargo new are always configured to use the latest stable edition.

There is a guide to rust editions at https://doc.rust-lang.org/edition-guide/editions/index.html

Reply:
Got it, thanks.